docs/adoption.md

```diff
@@ -17,39 +19,243 @@ Clone/fork the following repositories:
 The latter 2 produce Nuget packages, which if made public, could remove the need to clone their repositories.
 
 ## Nugets
+
 [HACT.Dtos](https://github.com/City-of-Lincoln-Council/HACT.Dtos) and [HousingRepairsOnline.Authentication](https://github.com/City-of-Lincoln-Council/HousingRepairsOnline.Authentication) produce Nuget packages which are consumed by other components.
 
 Each repository contains setup instructions which can be followed to establish connectivity.
 
 ## Infrastructure
+
 Azure is the supported Cloud platform.
 
 Create the following resouces in Azure:
 
 ### APIs
-- Azure app service plan 
+
+- Azure app service plan
   - Consider which service plan would best suit the needs. Premium V2 (P1v2: 1) is recommended
   - Use a region which is geographically close to the users/residents and the 3rd party data
 - Azure app service for each API
   - Create a 'staging' deployment slot
   - Each app service (and it's deployment slot) will have a publish profile which needs to be added to the repository secrets[^1]
 - Cosmos DB
-    - 2 databases - production and staging
-    - Each with `repair-requests` collections
+  - 2 databases - production and staging
+  - Each with `repair-requests` collections
 - Blob storage account
-    - Create 2 containers - production and staging
+  - Create 2 containers - production and staging
 
 ### Frontend
+
 - Create Azure Static Web App
   - A deployment token will need to be added to the repository secrets[^1]
 
+## Deploying to Azure
```

Comment:
Please add a note about using Infrastructure as Code and specifically Terraform before the steps to create resources.
This will provide context to the steps.